spot-ml/src/main/scala/org/apache/spot/proxy/ProxyWordCreation.scala

```diff
@@ -86,6 +115,12 @@ object ProxyWordCreation {
     }
   }
 
+  /**
+    * Defines if a domain is safe or not based on a known domain
+    *
+    * @param domain domain name
+    * @return
+    */
   def domainBelongsToSafeList(domain: String) = domain == "intel" // TBD parameterize this!
```

Comment:
this returns a Booelan - the return type should be noted in the signature and the @return parameter should informally describe the result